### ISSUE TITLE
[Adaptive cards -Food Order With Validation : Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8773]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/samples/FoodOrderWithValidation.html
OS Version: 23H2(OS Build 25997.1000)
Browser Version: 
Tool: Accessibility Insight For Web

**Repro Steps:**
1. Open the above URL on EDGE Dev browser.
2. Run the tool and observe the issue.

**Actual Result:**
Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds.  Currently color contrast ratio for the 'Steak, Chicken and Tofu' is 4.4.1 which is less than 4.5:1.

**Expected Result:**
Color Contrast ratio for the 'Steak, Chicken and Tofu' should be equal or greater than 4.5:1.

**Element path:**
 button[aria-label="Tofu"] > div

**Snippet:**
 <div style="overflow: hidden; text-overflow: ellipsis; white-space: nowrap;">Tofu</div>

**How to fix:** 
Fix any of the following:
  Element has insufficient color contrast of 4.41 (foreground color: #0079db, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

**User Impact:**
User with low vision will face difficulties if the contrast ratio of any text is less than 4.5:1.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

![WCAG1 4 3-Color contrast ratio for the Steak, Chicken and Tofu is 4 413 1 which is less than 4 5 1](https://github.com/microsoft/AdaptiveCards/assets/95913748/4ea85c68-3927-4a8b-b4c9-46ebc42a3504)

![WCAG1 4 3- Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds for the tofu text](https://github.com/microsoft/AdaptiveCards/assets/95913748/79c6b76a-dcfd-4883-8e36-60744fa8fa30)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Closing this issue as of now this issue is not repro.

![image](https://github.com/user-attachments/assets/cd37e4d6-dc8c-4d5f-8c28-c22a8c3dca9f)
